go.mod

```diff
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/Microsoft/go-winio v0.4.12
-	github.com/Microsoft/hcsshim v0.0.0-20190404205635-cf1c2137fae1
+	github.com/Microsoft/hcsshim v0.8.6
```

Comment:
Why this? This is wrong since the commit is much newer than the relase

Reply:
Sorry, updated!

Reply:
I'm not against this change I'm just wondering why we are doing it? Was this needed for these changes?

Reply:
I don't think so, was just updating since it appeared to have a fairly old commit for the shim. I can revert if needed.

Reply:
Its ok. In the future lets try to make these as separate commits because it makes it easier to find regressions.

Reply:
Will do!